policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/service/KafkaHealthIndicator.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.palisade.service.policy.service;
+
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.ConsumerGroupDescription;
+import org.apache.kafka.common.ConsumerGroupState;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+
+import uk.gov.gchq.palisade.service.policy.stream.ConsumerTopicConfiguration;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * Kafka health indicator. Check that the consumer group can be accessed and is registered with the cluster,
+ * if not mark the service as unhealthy.
+ */
+public class KafkaHealthIndicator implements HealthIndicator {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(KafkaHealthIndicator.class);
+
+    private String groupId;
```

Comment:
Remove `groupId` for thread safety.

Reply:
This class is a copy and paste from the work done in the attribute-masking-service.  For whatever reason, the annotation  @Value("${akka.kafka.consumer.kafka-clients.group.id}") is not showing in the class.